efit/get_data_from_EFIT_for_SOFT.py

```diff
@@ -0,0 +1,359 @@
+''' 	The purpose of this script is to get data from EFIT
+	specifically for SOFTv2 simulations
+	and write to an hdf5 file
+	See https://soft2.readthedocs.io/en/latest/magnetic_field.html#numerical-magnetic-field
+	Author: Alex Tinguely 230817
+	Usage: python3 get_data_from_EFIT_for_SOFT.py /path/to/gfile [/path/to/wallfile]
+'''
+
+### Import ###
+
+import numpy as np
+from matplotlib import pyplot as plt
+import scipy.interpolate as interp
+from scipy.optimize import curve_fit
+from shapely.geometry import Point
+from shapely.geometry.polygon import Polygon
+from copy import *
+import h5py
+import sys
+import geqdsk
+
+
+### Setup ###
+
+hdf5 		= 1;					# write to hdf5 file
+plot 		= 0;					# generate plots
+verbose 	= 1; 					# print to terminal
+read 		= 0; 					# try reading written file
+itot		= 1; 					# calculate total plasma current from Jphi
+ienc		= 1; 					# calculate enclosed plasma current from Br,Bz at LCFS

```

Comment:
One thought here is that we could also use the ``argparse`` module to set these options on the command-line, instead of having to edit this file every time we want a different configuration. By doing something like
```python
import argparse
...
parser = argparse.ArgumentParser("Export data from EFIT to SOFT equilibrium file")

parser.add_argument('-h', '--no-hdf5', dest='hdf5', help='Do NOT write EFIT data to HDF5 file', action='store_true', default=True)
parser.add_argument('-p', '--plot', help="Generate plots", action='store_true', default=False)
...
parser.add_argument('gFilename', help="Name of EFIT Gfile to load equilibrium from.")
parser.add_argument('-w', '--wall', dest="wallfile", help="Name of file containing wall.", nargs='?', default=None)
...
args = parser.parse_args()
...
# Using parameters
gfile = GEQDSK(args.gFilename)

if args.wallfile is not None:
    wall, RFW, ZFW = get_wall(args.wallfile)
else:
    # get from Gfile
    ...

if args.hdf5:
    # Save to HDF5 file...
    ...
```
The ``argparse`` module will then generate help for the script, and allow us to run it as e.g.
```
$ ./get_data_from_EFIT_for_SOFT.py gFilename -w wallFilename -p
```
Just a thought...